src/sphinxcontrib/towncrier/ext.py

```diff
@@ -137,10 +137,10 @@ def _lookup_towncrier_fragments(  # noqa: WPS210
         fragment_directory = None
 
```

Comment:
Looks like this may need a compat shim.

Reply:
It changes only the arg name, but not their order

Reply:
Alright, looking at this closer, it seems like the readability isn't compromised by the missing kwargs.